### DISCUSSION TITLE
Can't install discord.py

### DISCUSSION THREAD

**Discussion by OursBlanc42** [Rapptz/discord.py#7316] [Q&A]
### Summary

(failed building wheel yarl and multidict)

### Reproduction Steps

Hi there.

I try to install discord.py followings the instructions here: https://discordpy.readthedocs.io/en/stable/intro.html
(Please find error log attached)

Seem to be an error due to the architecture of my hardware (helios64 server) but I saw that we could have discord.py on raspberry which share the same type of architecture (ARM CPU) ... Strange ?

### Minimal Reproducible Code

```python
Failed building wheel for yarl

 
Failed building wheel for multidict


Command "/usr/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-tq3xlnpm/multidict/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-pxggph2c/install-record.txt --single-version-externally-managed --compile --user --prefix=" failed with error code 1 in /tmp/pip-install-tq3xlnpm/multidict/
```

### Expected Results

Install and use discord.py (for the moment if i run my discord bot, the line "import discord" not working : ModuleNotFoundError: No module named 'discord' )

### Actual Results

Please find attached actual results
[discord py log](https://github.com/Rapptz/discord.py/files/6861000/discord.py.log)

### Intents

void

### System Information

Python version : Python 3.7.3
OS : Armbian 21.05.1 Buster

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

**Comment by gogurtenjoyer** [comment]
It looks like there aren't pre-existing wheels for those requisites and so is attempting to build them locally. You'll want to have the `build-essential` package installed, which I think you do already, but also you need the development headers for python, which are missing according to the error line in the log: 
`   #include "Python.h"`
So, you'll want to install the `-dev` package associated with the python that you've got installed (such as `python3-dev`).
